lib/generator.js

```diff
@@ -80,7 +80,7 @@ module.exports = BasePlugin.extend({
 
           // replace capturing group with value
           var qval = (params.hasOwnProperty(key) && params[key] !== null) ? params[key] : param['default'];
```

Comment:
This was originally meant to prevent empty query params from being on the url.  

So this is more explicit in dealing with types than checking undefined.

```
qval = qval.toString()
if (qval != "")
```